src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,53 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import * as utils from 'util';
+import { IActionContext } from 'vscode-azureextensionui';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
+const configurePipelineCommand = 'configure-cicd-pipeline';
+const browsePipelineCommand = 'browse-cicd-pipeline';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
```

Comment:
💡 🔂 Repetitive duplicate function: both `configurePipeline` and `browsePipeline` are `duplicate` functions, i.e. it is repetitive code and only thing changed is the `const, we can merge the 2 functions with better name, and supply that via `register`. = )

Another question for you guys is: It might also be worth to share the thought process behind this change for both Kubernetes and aks services extension.